### DISCUSSION TITLE
The blog is full of code instead of ordinary blog interface

### DISCUSSION THREAD

**Discussion by znqi** [academicpages/academicpages.github.io#2301] [Q&A]
Do please visit https://znqi.github.io/year-archive/(my personal homepage which is constructed on "academicpages" repo) and you can find a blog called"双摆运动的数值解法“, which was tried by myself after having edited and uploaded the markdown file(2024-08-04-doublependulum.md). The blog is full of code instead of an ordinary blog interface
<img width="1304" alt="截屏2024-08-04 22 07 51" src="https://github.com/user-attachments/assets/7357482c-4a25-4b9c-ae18-030d04b6f599">
Could anyone teach me how to fix it?
Also here's my repository on github. 
https://github.com/znqi/znqi.github.io
If it is convenient, I sincerely ask you to help me check whether there are any errors in editing the files in the repository.

**Comment by rjzupkoii** [comment]
It looks like the file is being served to the web browser incorrectly and what stands out to me is that the `permalink` field still has the `.md` extension in it, try updating it to the following:

```markdown
permalink: /2024/08/04/doublependulum
```

**Reply by znqi** [reply]
Thank you so much. It's solved!
